packages/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp

```diff
@@ -29,7 +29,12 @@
 
 #include <DefaultComponentsRegistry.h>
 #include <DefaultTurboModuleManagerDelegate.h>
+#if __has_include("<autolinking.h>")
+#define AUTOLINKING_AVAILABLE 1
```

Comment:
Nit: Technically, autolinking is still available. Just not TurboModules.

Reply:
the `define` is just to have a quick way to compile out the pieces that could not be available in the rest of the file.

Reply:
No, I know. I was commenting on the name of the macro 😄